src/v/redpanda/admin/api-doc/debug_bundle.def.json

```diff
@@ -52,6 +52,9 @@
     },
     "tls_insecure_skip_verify": {
       "type": "boolean"
+    },
+    "namespace": {
```

Comment:
This results in a compilation failure.

We don't actually need this type. It could be inlined into the schema, which should prevent the code from being generated.